types/src/ledger/log.rs

```diff
@@ -121,6 +123,28 @@ impl VerifiableLog {
             })
     }
 
+    fn sort_attestations_by_timestamp(&self) -> Vec<&TimestampedHeadlessAttestation> {
+        self.pod_metadata
+            .attestations
+            .iter()
+            .sorted_by_key(|a| a.timestamp.as_seconds())
+            .collect()
+    }
+
+    pub fn get_sorted_attestation_timestamps(&self) -> Vec<Uint<256, 4>> {
```

Comment:
NIT: in other places we use `U256` alias. Can you use it too here?
```suggestion
    pub fn get_sorted_attestation_timestamps(&self) -> Vec<U256> {
```

Additionally, the function declaration does not specify whether the returned values are in seconds or microseconds, making it unclear without reading the source. How about inlining this function where it is needed instead of putting it in the SDK?

Reply:
I think this function would help users verify certificates on Ethereum by making the users not have to deal with conversions, I will update the name of the function though.